src/commands/moderation/prefixCommands/functions/deleteChannelDefaultVersion.ts

```diff
@@ -0,0 +1,88 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommandChannelDefaultVersion, Logger, makeEmbed, clearSinglePrefixCommandChannelDefaultVersionCache } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Unset Default Channel Version - No Connection',
+    description: 'Could not connect to the database. Unable to unset the default channel version.',
+    color: Colors.Red,
+});
+
+const failedEmbed = (channel: string) => makeEmbed({
+    title: 'Prefix Commands - Unset Default Channel Version - Failed',
+    description: `Failed to unset the default channel version with for ${channel}.`,
+    color: Colors.Red,
+});
+
+const doesNotExistsEmbed = (channel: string) => makeEmbed({
+    title: 'Prefix Commands - Unset Default Channel Version - Does not exist',
+    description: `The default channel version with for ${channel} does not exists. Can not unset it.`,
+    color: Colors.Red,
+});
+
+const successEmbed = (channel: string) => makeEmbed({
+    title: `Default channel version for channel ${channel} was unset successfully.`,
+    color: Colors.Green,
+});
+
+const modLogEmbed = (moderator: User, channel: string) => makeEmbed({
+    title: 'Prefix Commands - Default Channel Version unset',
+    fields: [
+        {
+            name: 'Channel',
+            value: channel,
+        },
+        {
+            name: 'Moderator',
+            value: `${moderator}`,
+        },
+    ],
+    color: Colors.Red,
+});
+
+const noModLogs = makeEmbed({
+    title: 'Prefix Commands - Unset Default Channel Version - No Mod Log',
+    description: 'I can\'t find the mod logs channel. Please check the channel still exists.',
+    color: Colors.Red,
+});
+
+export async function handleDeletePrefixCommandChannelDefaultVersion(interaction: ChatInputCommandInteraction<'cached'>) {
+    await interaction.deferReply({ ephemeral: true });
+
+    const conn = getConn();
+    if (!conn) {
+        await interaction.followUp({ embeds: [noConnEmbed], ephemeral: true });
+        return;
+    }
+
+    const channel = interaction.options.getChannel('channel')!;
+    const moderator = interaction.user;
+
+    //Check if the mod logs channel exists
+    const modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS) as TextChannel;
+    if (!modLogsChannel) {
```

Comment:
```suggestion
    let modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS);
    if (!modLogsChannel || !modLogsChannel.isTextBased()) {
        modLogsChannel = null;
```
Might be safer to actually check if the channel is a text channel? Especially if we want to offer the codebase to other projects. Then, if it's not a text channel set it to null to prevent accidental attempts to send into a non-text channel.